pkg/services/helpers/fsutil/blockstorage.go

```diff
@@ -0,0 +1,55 @@
+package fsutil
+
+import (
+	"crypto/sha256"
+	"time"
+)
+
+// Constants related to NeoFS block storage.
+const (
+	// OIDSize is the size of the object ID in NeoFS.
+	OIDSize = sha256.Size
+	// DefaultTimeout is the default timeout for NeoFS requests.
+	DefaultTimeout = 5 * time.Minute
```

Comment:
Either adjust `Timeout: 10m` in the `./config/protocol.*.yml` or change this default according to the config.

Reply:
Should be done in a separate commit (or in the first commit).

Reply:
rebase artefact -sorry, I will recheck it.